### ISSUE TITLE
[Question]How to pre-process multi small target labels to multi bboxes?

### ISSUE THREAD

**Issue by wushu526** [MIC-DKFZ/nnDetection#117]
### :question: Question

I find that I get big bbox after running 'nndet_prep xxx' using my own dataset (multi small liver tumor). These large boxes should actually enclose all the small targets. What I wish to get is multi small bbox at every image. Also, I got some BIG bbox after training and predicting in this way. Actually, the purpose is detect the small tumor (predicting small box).

**Comment by mibaumgartner** [comment]
Dear @wushu526 ,

nndetection will determine the bounding boxes from the labels that were provided by the dataset. They will enclose the provided objects as close as possible (1 pixel distance to the segmentation boarder to be precise).

From what I understand, the problem is that the preprocessed bounding boxes encloses several smaller lesions which indicates a mistake in the ground truth labels. If every lesion has its own label (i.e. lesion 1 has label 1, lesion 2 has label 2 etc.) nnDetection will derive one box per lesion. Please refer to the `Label Format` section of the readme for more info (make suer you don't confuse the annotation format with semantic segmentation where all lesions will have the same label!)

Best,
Michael

**Comment by wushu526** [comment]
Dear @mibaumgartner ,

Thanks a lot! I have already solved the problem by using different values of labels. 
BUT, I got another problem. Sometime I can not predict imagesTs after I starting training. Also, there are no file in the folder _preprocessed/D3V001_3d/imagesTs/_. How can I preprocess the testing set and predict them?

The comands I used: 
`nndet_predict 111 RetinaUNetV001_D3V001_3d --fold 0`
`nndet_predict 111 RetinaUNetV001_D3V001_3d --fold 0 --no_preprocess `
`nndet_predict 111 RetinaUNetV001_D3V001_3d --fold 0 --no_preprocess --test`
